 website2/_header.scss

```diff
@@ -0,0 +1,127 @@
+.header 
+{
+  
+  background-color: #1F2326;
+  margin: auto;
+  color: #ffffff;
+
+
+  &-links 
+  {
+    
+    margin-left: 150px;
+    margin-right: 150px;
+   font-family: "Open Sans", Arial, Helvetica, sans-serif;
+    font-weight: lighter;
+    font-size: small;
+   justify-content: space-between;
+
+     ul
+    {
+
+      display: flex;
+      list-style: none;
+      padding: 0px;
+      margin-top: 0px;
+      margin-bottom: 0px;
+      
+      
+    }
+   
+  }
+}
+
+
+.header-links ul li
+{
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.link-holder a
+{
+  text-decoration: none;
+  text-decoration: none;
+  color: #ffffff;
+  font-family: "Open Sans", Arial, Helvetica, sans-serif;
+  line-height: 25px;
+}
+
+.link-holder a:hover
+{
+  color: #0096d2;
+}
+
+.link-holder 
+{
+  width: 1000px;
+}
+
```

Comment:
write above 3 selectors in sass way
.link-holder a
.link-holder a:hover
+.link-holder